### ISSUE TITLE
Loss function for binary classification

### ISSUE THREAD

**Issue by manuel-88** [facebookarchive/caffe2#629]
Is there a loss function for binary classification which I can use with the CNNModelHelper?

**Comment by peterneher** [comment]
I think you should use ModelHelper instead of cnn.CNNModelHelper.
For binary classification you can simply use LabelCrossEntropy and softmax with two outputs. But there is also a method called MakeTwoClass in https://github.com/caffe2/caffe2/blob/master/caffe2/operators/cross_entropy_op.cc but I dint't try to use it.
